package.json

```diff
@@ -4,10 +4,10 @@
   "private": true,
   "scripts": {
     "prepare": "husky install",
-    "hh:compile": "npx hardhat compile && npx hardhat deploy --network hardhat --tags underlying,oracle,registryToken,vault,market",
+    "hh:compile": "npx hardhat compile && npx hardhat deploy --network hardhat --tags underlying,oracle,registryToken,vault,vaultTimeLock,market",
```

Comment:
`vaultTimeLock` tag no longer used so we can remove it from here and the other deployment commands